tools/slurm/scripts/slurmcluster.sh

```diff
@@ -214,16 +206,6 @@ if [[ $OPT_CONTAINER_RUN_TYPE == "enroot" ]]; then
     fi
 fi
 
-TEMPYAML=$TEMPDIR/slurmcluster.yaml
```

Comment:
I think on line 143 there is another reference to ```$DETERMINED_AGENT```

Reply:
I didn't remove the line in 143 because the description of those lines meant grabbing token for launcher service, which I think is still valid. 
Though when we do not set `DETERMINED_AGENT=1` anymore, hence those set of lines will never be executed.

I am inclined towards removing the line 143 `if [[ -z $DETERMINED_AGENT ]]; then` if condition and bring the lines within line 144 - 150 outside the if parameter.

What do you think?

Reply:
I think since we don't set ```DETERMINED_AGENT```

the ```if [[ -z $DETERMINED_AGENT ]]; then``` checks that ```DETERMINED_AGENT``` isn't set so the if will always be true. So I think your suggestion of removing the if statement and keeping the rest of the code is right